tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -80,28 +79,17 @@ def generate(schema, error_category, error_message, record=None, field=None):
     return generate
 
 
-class SchemaManager:
-    """Manages one or more RowSchema's and runs all parsers and validators."""
+def value_is_empty(value, length):
+    """Handle 'empty' values as field inputs."""
+    empty_values = [
+        '',
```

Comment:
include `''` (blank string) in the `value_is_empty` list - this manages the effect of python returning `''` when getting an out-of-bounds substring